homeassistant/components/watergate/coordinator.py

```diff
@@ -0,0 +1,44 @@
+"""Coordinator for Watergate API."""
+
+from datetime import timedelta
+import logging
+
+from watergate_local_api import WatergateApiException, WatergateLocalApiClient
+from watergate_local_api.models import DeviceState
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WatergateAgregatedRequests:
+    """Class to hold aggregated requests."""
+
+    def __init__(self, state: DeviceState | None = None) -> None:
+        """Initialize aggregated requests."""
+        self.state = state
```

Comment:
As you remember, I've already prepared this coordinator for the rest of the data that we are going to fetch. I will keep it since I will be building on it in the near future.

Reply:
I am missing the point why we should do this? We work incrementally and thus should only have code that is required for the valve. And then this indeed is a wrapper